intro-HPC/ch_connecting.tex

```diff
@@ -370,8 +370,8 @@ \section{First Time connection to the HPC infrastructure}
    cluster - full - free -  part - total - running - queued
              nodes  nodes   free   nodes    jobs      jobs
 -------------------------------------------------------------------
-    golett    168      2     16     200      N/A       N/A
-    phanpy     15      0      1      16      N/A       N/A
+     doduo    168      2     16     200      N/A       N/A
```

Comment:
kirlia?

Reply:
Order is also different in the actual output (old to new), we should fix that too...

Reply:
fixed